app/main/controllers/template/rtmedia-functions.php

```diff
@@ -3926,3 +3934,19 @@ function rtmedia_show_title() {
 	}
 }
 
+
+/**
+ * Checks if BuddyPress is activated or not.
+ *
+ * Will Check that if the BuddyPress plugin is activated or not.
```

Comment:
It's duplicate line. Remove this line. `Checks if BuddyPress is activated or not.` is enough.